ivy/stateful/module.py

```diff
@@ -1079,7 +1079,7 @@ def _forward(self, *a, **kw):
         params_hk = self._dict_to_hk_flat_map(self.v.cont_to_dict())
         ret = self._native_module.apply(params_hk, 0, *a, **kw)
         if isinstance(ret, tuple):
-            return ivy.args_to_native(*ret)
+            return ivy.args_to_native(*ret)[0]
```

Comment:
just out of curiosity, why are we even checking if `ret` is a `tuple` and using `args_to_native` if that's the case? Ideally `ivy.to_native` should be able to correctly convert arrays in the tuple to native,
```
import ivy

ivy.set_backend("numpy")
x = ivy.array([1., 2., 3.])
print(ivy.to_native((x,), nested=True))
```

Reply:
@vedpatwardhan Yeah I mean I didn't add that logic 😅 Although my thinking was the same on first glace. Probably look at the line history to see who authored this originally

Reply:
It's been quite some time since I last reviewed this code, and I don't recall every minor detail. However, upon examining the implementation, it appears to be an extraneous addition. As @vedpatwardhan suggested, we can effectively substitute this logic with `ivy.to_native(ret, nested=True)`.

Reply:
Sounds good! Thanks for making the changes @hmahmood24 and the helpful suggestions @RickSanchezStoic @YushaArif99. I'll resolve this now